src/csharp/Microsoft.Spark.E2ETest/IpcTests/BroadcastTests.cs

```diff
@@ -49,15 +52,45 @@ public void TestMultipleBroadcastWithoutEncryption()
 
             string[] actual = ToStringArray(_df.Select(udf(_df["_1"])));
             Assert.Equal(expected, actual);
+            bc1.Destroy();
+            bc2.Destroy();
+        }
+
+        /// <summary>
+        /// Test Broadcast support by broadcasting a large (>100MB) object.
+        /// </summary>
+        [Theory]
+        [InlineData("true")]
+        [InlineData("false")]
+        public void TestLargeBroadcastValue(string isEncryptionEnabled)
```

Comment:
I don't think we need to include this unless it's testing uncovered code path? (I think I asked to test this scenario just for a sanity check and probably for a previous implementation).

Reply:
Removed.